### ISSUE TITLE
Helm version not bumped to 3.4.1?

### ISSUE THREAD

**Issue by KokXinTan** [thingsboard/thingsboard-ce-k8s#67]
As per title, can Helm version be updated to 3.4.1?

Thanks

**Comment by d-lopes** [comment]
Yes please, i would also be interested in this. Apparently, everything is already there in this directory: https://github.com/thingsboard/thingsboard-ce-k8s/tree/master/helm

But the latest version on artifacthub.io is only 0.1.2-rc1 which points to the older release 3.3.4.1: https://artifacthub.io/packages/helm/thingsboard/thingsboard/0.1.2-rc.1 

Can you please make the newest helm version available there?

Thanks & Best regards
 Dominique

**Comment by Raven888888** [comment]
Bump

**Comment by midokura-agustin** [comment]
You can anyway install latest version specifying the image tag with something like:
```
helm install RELEASENAME HELMCHART -n NAMESPACE --set global.image.tag="3.4.3"
```
I'll send a PR doing a version bump and see if it's accepted

**Comment by midokura-agustin** [comment]
https://github.com/thingsboard/thingsboard-ce-k8s/pull/82
